src/scripts/main.js

```diff
@@ -1 +1,24 @@
 'use strict';
+
+const nav = document.querySelector('.list-item');
+const imgLarge = document.querySelector('.gallery__large-img');
+const a = document.querySelector('a');
+const img = document.querySelector('.gallery__thumb');
+
+a.addEventListener('click', () => {
+  event.preventDefault();
```

Comment:
The `event` object is not defined within the arrow function. You should pass `event` as a parameter to the function to access it.